In-house patch to remove unneeded JS libraries.
This patch is not suitable for pushing upstream.

--- horizon-9.0.1/horizon/site_urls.py.~1~	2016-06-02 13:05:56.000000000 -0700
+++ horizon-9.0.1/horizon/site_urls.py	2016-06-29 23:54:56.842020230 -0700
@@ -22,8 +22,6 @@ from django.conf.urls import patterns
 from django.conf.urls import url
 from django.views.generic import TemplateView  # noqa
 
-from horizon.test.jasmine import jasmine
-
 urlpatterns = patterns(
     'horizon.views',
     url(r'^home/$', 'user_home', name='user_home')
@@ -40,12 +38,3 @@ urlpatterns += patterns(
         name="set_language"),
     url(r'^i18n/', include('django.conf.urls.i18n'))
 )
-
-if settings.DEBUG:
-    urlpatterns += patterns(
-        '',
-        url(r'^jasmine-legacy/$',
-            TemplateView.as_view(
-                template_name="horizon/jasmine/jasmine_legacy.html"),
-            name='jasmine_tests'),
-        url(r'^jasmine/.*?$', jasmine.dispatcher))
--- horizon-9.0.1/openstack_dashboard/static_settings.py.~1~	2016-06-02 13:05:56.000000000 -0700
+++ horizon-9.0.1/openstack_dashboard/static_settings.py	2016-06-29 23:54:56.842651790 -0700
@@ -31,7 +31,6 @@ import xstatic.pkg.bootswatch
 import xstatic.pkg.d3
 import xstatic.pkg.font_awesome
 import xstatic.pkg.hogan
-import xstatic.pkg.jasmine
 import xstatic.pkg.jquery
 import xstatic.pkg.jquery_migrate
 import xstatic.pkg.jquery_quicksearch
@@ -82,9 +81,6 @@ def get_staticfiles_dirs(webroot='/'):
         ('horizon/lib/font-awesome',
             xstatic.main.XStatic(xstatic.pkg.font_awesome,
                                  root_url=webroot).base_dir),
-        ('horizon/lib/jasmine',
-            xstatic.main.XStatic(xstatic.pkg.jasmine,
-                                 root_url=webroot).base_dir),
         ('horizon/lib/jquery',
             xstatic.main.XStatic(xstatic.pkg.jquery,
                                  root_url=webroot).base_dir),
